packages/web3-eth/src/rpc_methods.ts

```diff
@@ -97,9 +88,11 @@ export async function getStorageAt(
 	storageSlot: Uint256,
 	blockNumber: BlockNumberOrTag,
 ) {
-	validateAddress(address);
-	validateHexStringInput(storageSlot);
-	validateBlockNumberOrTag(blockNumber);
+	validator.validate(
+		['address', 'hex', 'blockNumberOrTag'],
+		[address, storageSlot, blockNumber],
+		{ silent: false },
```

Comment:
```suggestion
```

This doesn't need to be included, right? `silent: false` is the default [here](https://github.com/ChainSafe/web3.js/blob/e21d55bd6cdc8c181289e8faaf9afa691508d2e1/packages/web3-validator/src/web3_validator.ts#L57):

![image](https://user-images.githubusercontent.com/20375610/148214318-e9008f78-b394-47cd-89f1-9c74c8c86c17.png)